prisma/schema.prisma

```diff
@@ -49,13 +51,36 @@ model Authenticator {
 }
 
 model Group {
-  id          String  @id @default(uuid())
+  id          String             @id @default(uuid())
   name        String
-  description String  @db.Text
+  description String             @db.Text
   imgUrl      String?
   imgAlt      String?
   events      Event[]
   users       User[]
+  socalLinks  GroupSocialLink[]
+}
+
+enum SocalLinkType {
+  FACEBOOK
+  TWITTER
+  INSTAGRAM
+  LINKEDIN
+  DISCORD
+  WEBSITE
+}
+model GroupSocialLink {
+  groupId String @id @unique
```

Comment:
Problem: if the `groupId` is unique in this table, then each group can only have one GroupSocialLink.

Maybe we can make the combination of `type` and `groupId` unique and the identifier? Or alternatively, just add an auto-created uuid id!